### PR TITLE
fix stiff translation for chinese

### PR DESCRIPTION
**Pull request by zouyee**

### CODE REVIEW

File: i18n/messages-zh.xtb [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
   <translation id="2248903848244811172" key="MSG_BREADCRUMBS_HORIZONTAL_POD_AUTOSCALERS_LABEL" desc="Label 'Horizontal Pod Autoscalers' that appears as a breadcrumbs on the action bar.">Pod水平自动伸缩</translation>
-  <translation id="5747298115200147575" key="MSG_BREADCRUMBS_INGRESSS_LABEL" desc="Label 'Ingress' that appears as a breadcrumbs on the action bar.">入口地址(Ingress)</translation>
+  <translation id="5747298115200147575" key="MSG_BREADCRUMBS_INGRESSS_LABEL" desc="Label 'Ingress' that appears as a breadcrumbs on the action bar.">路由(Ingress)</translation>
   <translation id="2741276106482217695" key="MSG_BREADCRUMBS_INTERNALERROR_LABEL" desc="Label for internal error page for breadcrumbs on the action bar.">内部错误(Internal error)</translation>
-  <translation id="2024511088483477440" key="MSG_BREADCRUMBS_JOBS_LABEL" desc="Label 'Jobs' that appears as a breadcrumbs on the action bar.">批量任务(Jobs)</translation>
+  <translation id="2024511088483477440" key="MSG_BREADCRUMBS_JOBS_LABEL" desc="Label 'Jobs' that appears as a breadcrumbs on the action bar.">批处理任务(Jobs)</translation>
```
Comment (chengji77) [inline i18n/messages-zh.xtb]: The previous one looks good enough. I'll prefer the shorter one.

File: i18n/messages-zh.xtb [hunk 2]
```diff
@@ -2,20 +2,20 @@
...
   <translation id="1744394420157932960" key="MSG_BREADCRUMBS_DEPLOYMENTS_LABEL" desc="Label 'Deployments' that appears as a breadcrumbs on the action bar.">部署(Deployments)</translation>
   <translation id="9213553703311647006" key="MSG_BREADCRUMBS_DEPLOY_APP_LABEL" desc="Breadcrum label for the deploy containerized app form view.">创建应用</translation>
   <translation id="3098067826555563293" key="MSG_BREADCRUMBS_DEPLOY_FILE_LABEL" desc="Breadcrum label for the YAML upload form">上传(Upload)</translation>
   <translation id="2248903848244811172" key="MSG_BREADCRUMBS_HORIZONTAL_POD_AUTOSCALERS_LABEL" desc="Label 'Horizontal Pod Autoscalers' that appears as a breadcrumbs on the action bar.">Pod水平自动伸缩</translation>
-  <translation id="5747298115200147575" key="MSG_BREADCRUMBS_INGRESSS_LABEL" desc="Label 'Ingress' that appears as a breadcrumbs on the action bar.">入口地址(Ingress)</translation>
+  <translation id="5747298115200147575" key="MSG_BREADCRUMBS_INGRESSS_LABEL" desc="Label 'Ingress' that appears as a breadcrumbs on the action bar.">路由(Ingress)</translation>
```
Comment (chengji77) [inline i18n/messages-zh.xtb]: The Chinese here means "routing" by literal. What's the context here?

**Comment by maciaszczykm** [comment]
@chengji77 Thank you for you review.

**Comment by maciaszczykm** [comment]
@zouyee Can you adress @chengji77 comments?

File: i18n/messages-zh.xtb (hunk 1 above)
Comment (zouyee) [inline i18n/messages-zh.xtb]: In my opinion，we could correct  `批量任务` to  `任务`.

File: i18n/messages-zh.xtb (hunk 2 above)
Comment (zouyee) [inline i18n/messages-zh.xtb]: In general， it is not necessary to translate all words which we could not find suitable words to express

**Comment by maciaszczykm** [comment]
@zouyee @chengji77 Is it ready to merge?

**Comment by zouyee** [comment]
👌

File: i18n/messages-zh.xtb (hunk 1 above)
Comment (maciaszczykm) [inline i18n/messages-zh.xtb]: @zouyee Was it resolved?

**Pull request merged**